### ISSUE TITLE
When will support the other datasets?

### ISSUE THREAD

**Issue by rann1018** [mihaidusmanu/local-feature-refinement#3]
Hi, @mihaidusmanu 

I find this platform really useful for testing local feature refinement methods, and thanks a lot for 
your good work.
I want to do some further research based on your code. However, now this platform only supports 
ETH3D triangulation task in master branch. 
I notice you said HPatches Sequences and ETH3D localization would be coming soon. Can you 
tell me when you will release the code to support these datasets?

Sincerely appreciate for your help.

**Comment by mihaidusmanu** [comment]
Hey. Sorry for the late reply. ETH3D localization benchmark is available on the eth3d-loc branch - I sadly did not have time to re-run everything to double check the results, but the code should be there and yielding similar results (I only refactored it a bit).

Regarding HPatches, we simply built on top of the D2-Net evaluation available at https://github.com/mihaidusmanu/d2-net/tree/master/hpatches_sequences. The code for two-view refinement is available in the following issue: https://github.com/mihaidusmanu/local-feature-refinement/issues/1.

I will try to clean up the HPatches code and add it to the repo in the following days!
